pmg/views.py

```diff
@@ -345,6 +346,18 @@ def committee_detail(committee_id):
     )
 
 
+@app.route('/committee/<int:committee_id>/follow-cte')
+@app.route('/committee/<int:committee_id>/follow-cte/')
+def committee_detail_follow_cte(committee_id):
+    follow_cte_post_registration_or_login(committee_id)
+    
+    # Remove confirmation email flash
+    session['_flashes'] = []
```

Comment:
We already show another message at the top of the page mentioning the confirmation email. The page looks very busy with all the banners, so I thought we could remove the login extensions confirmation message.

I'm not sure if this is the best way to do it though?